doc/md/fundamentals/2-types/3-functions.md

```diff
@@ -72,7 +82,7 @@ let num = identity<Nat>(42);
 let text = identity<Text>("Hello");  
 ```
 
-The type of identity is `<T>T -> T`, indicating that when supplied with any type `T`, and argument of type `T`, it returns a result of the same type `T`.
```

Comment:
Come to think of it, a more interesting example might be `swap<T, U>(t : T, u:U) : (U, T) { (t, u) }` because it actually does something useful.